minidump/src/minidump.rs

```diff
@@ -912,26 +912,31 @@ impl Module for MinidumpModule {
 
     fn code_identifier(&self) -> CodeId {
         match self.codeview_info {
-            Some(CodeView::Elf(ref raw)) => CodeId::from_binary(&raw.build_id),
-            Some(CodeView::Pdb70(ref raw)) if raw.age == 0 => {
-                // This is a macOS minidump. It looks like the age is typically
-                // >0 for Windows minidumps, and Breakpad sets it to 0 on macOS.
-                // We're relying on this observation in order to distinguish
-                // between these two platforms since macOS uses PDB70 instead of
-                // its own dedicated format.
-                // See the following linked comment in a Windows PDB writer:
-                // https://github.com/microsoft/microsoft-pdb/blob/082c5290e5aff028ae84e43affa8be717aa7af73/PDB/dbi/pdb.cpp#L602-L606
-                // TODO: Figure out a way to get access to `MinidumpSystemInfo`
-                // here instead of relying on `age`.
-                CodeId::new(format!("{:#}", raw.signature))
-            }
-            _ => {
-                // TODO: Breakpad stubs this out on non-Windows.
```

Comment:
I'm not too sure what exactly this meant originally but I _did_ remove this in the PR, since we have relatively well-defined behaviours for all three known `CodeView` signatures as well as those that are unknown. Would like to double check to make sure this is correctly addressed and therefore fine to remove.